src/test/spec/unified_runner/mod.rs

```diff
@@ -236,3 +239,10 @@ async fn test_examples() {
     let _guard = LOCK.run_exclusively().await;
     run_spec_test(&["unified-runner-examples"], run_unified_format_test).await;
 }
+
+#[cfg_attr(feature = "tokio-runtime", tokio::test)]
+#[cfg_attr(feature = "async-std-runtime", async_std::test)]
+async fn test_versioned_api() {
+    let _guard = LOCK.run_exclusively().await;
```

Comment:
We usually annotate the type on these (i.e. `let _guard: RwLockWriteGuard<_> = LOCK.run_exclusively().await`) because the compiler doesn't generate any warnings/errors if you forget to `await` without the annotation. It's not an issue here but it would be good to add it in anyway for the sake of consistency with other tests.

Reply:
That was a shameless copy/paste from the test above this one. I'll update both methods.